### PR TITLE
Enoble/unity ux

### PR DESCRIPTION
**Pull request by fr0hsty**
This should not be approved until https://euclideon.com/unity has been updated to display both the .unitypackage download link as well as quick-start instructions for using it.

This branch is essentially the original state of master with the addition of a VDK toolbar menu within unity  allow users to set their VDK username/password within the editor rather then modifying .cs files directly.

It also includes the unity package manifest .json file to force the dependency on post processing to enable by default, removing the chance of nullrefrence errors on initial setup.

Allow unsafe code has also been force enabled by default to prevent a secondary potential error.

In short, this branch is supposed to be a quality of life change for new unity users to quickly get setup via git, or via .unitypackage download.

### CODE REVIEW

File: integrations/unity-csharp/Assets/VDK/VDKUserInfoEditor.cs [hunk 1]
```diff
@@ -0,0 +1,143 @@
+﻿using UnityEngine;
+using System;
+using UnityEditor;
+using UnityEditor.PackageManager;
+using UnityEditor.PackageManager.Requests;
+using Vault;
+
+
+
+
```
Comment (OsiJr) [inline integrations/unity-csharp/Assets/VDK/VDKUserInfoEditor.cs:10]: So much whitespace

**Review by OsiJr — APPROVED** [review]

File: integrations/unity-csharp/Assets/VDK/VDKUserInfoEditor.cs (hunk 1 above)
Comment (fr0hsty) [reply integrations/unity-csharp/Assets/VDK/VDKUserInfoEditor.cs:10]: Ahh cheers, I'll make sure to clear that out in future work ty
